### PR TITLE
Changes for Dart 2.0

### PR DESCRIPTION
**Pull request by enyo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.3.0
+- Dar 2.0 support
```
Comment (mit-mit) [inline CHANGELOG.md]: Dart, not Dar

Comment (enyo) [reply CHANGELOG.md]: Thanks. Fixed.

**Comment by andresaraujo** [comment]
Thanks @enyo will take a look tomorrow, and release a version.

FYI I'm working v2 of this library, it will be a lot simpler and will not depend on package `intl` so the code will be the same for browser and server. It will be ready in a few days

**Comment by enyo** [comment]
Thanks

**Pull request merged**